lib/ts/supertokens.ts

```diff
@@ -39,7 +39,7 @@ export default class SuperTokens {
         }
 
         this.recipeList = config.recipeList.map((recipe) => {
-            return recipe(this.appInfo);
+            return recipe(this.appInfo, config.storageHandlers);
```

Comment:
Where is `config. storageHandlers` getting normalised? And why is it not getting normalised in here itself? Just like how appInfo is getting normalised here itself.